server/src/utils/fight/fightMethods.ts

```diff
@@ -1772,33 +1777,14 @@ export const playFighterTurn = (
         updateStats(stats, currentFighter.id, 'consecutiveThrows', 1);
         checkAchievements(stats, achievements);
 
-        // Check if opponent blocked (harder than melee)
-        if (!deflected && block(currentFighter, currentOpponent, 2)) {
-          damage = 0;
-
-          // Add block step
-          fightData.steps.push({
-            a: StepType.Block,
-            f: currentOpponent.id,
-          });
-
-          // Update block stat
-          updateStats(stats, currentOpponent.id, 'blocks', 1);
-          updateStats(stats, currentOpponent.id, 'consecutiveBlocks', 1);
-          checkAchievements(stats, achievements);
-        } else {
-          // Reset block stat
-          updateStats(stats, currentOpponent.id, 'consecutiveBlocks', 0);
-        }
-
```

Comment:
Please don't remove my hard work D:

Reply:
it was in double o_O

Reply:
Oh I mix myself , who was block the other was evade ^^ that's on me